npm/test/db/db.test.ts

```diff
@@ -475,18 +478,30 @@ tap.test('dbs', async () => {
       t.same(ret3.data, []);
       t.same(ret4.data, []);
     });
+
+    // tap.test('close(): ' + dbEngine, async () => {
+    //   for (const [, value] of Object.entries(dbObjs)) {
+    //     await value.close();
+    //   }
+    // });
```

Comment:
Wanna remove this?

Reply:
No, this has to become part of the test. Currently makes the test fail.